### PR TITLE
[DetailsList] Update examples to clarify columns

### LINKED ISSUE #8997: DetailsList thumbnails don't render on implementation page.
On the implementation of customgroupheaders, the thumbnails don't render and instead only show their text links (ie. //placehold.it/221x221). 

https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/customgroupheaders

I'm using Chrome Version 74.0.3729.131 (Official Build) (32-bit) on Windows 10.

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #8997
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Update column names in examples.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9005)

### CODE REVIEW

**Comment by jdhuntington** [comment]
I'm in favor of option 1 - I think the examples should be straightforward and use as little code as possible. I worry that we will encourage people to overcomplicate implications if all examples lean on custom cell rendering.

File: packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx [hunk 1]
```diff
@@ -110,4 +104,16 @@ export class DetailsListCustomGroupHeadersExample extends React.Component<{}, {}
       props!.onToggleCollapse!(props!.group!);
     };
   }
+
+  /** alterColumnNames modifies columns from example data to clarify their usage in examples.
+   * It is not necessary to replicate this function in your own code.
+   */
+  private _alterColumnNames = (cols: IColumn[]): IColumn[] => {
+    cols.forEach(c => {
+      if (c.name === 'thumbnail') {
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx]: Can't you just change the column name in the example data instead? (then fix anything that breaks)

Alternatively you could either fix the thumbnail column to have a proper renderer, or add code to hide it in most circumstances.

Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx]: I explored providing a consistent renderer for thumbnails across our examples, but it changes our more tabular `DetailsList` examples too much.

image

We could make the thumbnails smaller, but at that point they would be comically small and unrealistic.

After noticing the above, I'm of the opinion that we minimize the cases were we showcase thumbnail rendering in our `DetailsList` examples.

Comment (ecraig12345) [reply packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.CustomGroupHeaders.Example.tsx]: Hmm okay, thanks for investigating. Similarly to what JD said, I'd be in favor of removing the column entirely (or *maybe* adding an option to include it, which is false by default).

**Comment by jdhuntington** [comment]
Going to close this, as I don't think it's the right solution in light of the discussion. Thanks all!

**Pull request closed without merging**